architectures/models/SSDNet.py

```diff
@@ -97,5 +97,7 @@ def forward(self, x):
         x = self.inv5(x)
         _1bbox, _1class = self.out5(x)
 
-        return [torch.cat([_20bbox, _10bbox, _5bbox, _3bbox, _2bbox, _1bbox], dim=1),
-                torch.cat([_20class, _10class, _5class, _3class, _2class, _1class], dim=1)]
+        # bboxes prediction:  B x (20*20) * 6 x 4 ...
+        # class prediction:  B x (20*20) * 6 x 3 ...
+        return [torch.cat([_10bbox, _5bbox, _3bbox, _2bbox, _1bbox], dim=1),
```

Comment:
So we prune the searching space by not considering smaller objects than ~10x10?

Reply:
Vehicles and people would rarely be a good fit for the 20x20 grid, so we ditch that as it basically only worsens class imbalance